front/app/components/FullscreenImage.tsx

```diff
@@ -18,13 +18,32 @@ const FullscreenImage = ({ src, altText }: Props) => {
   const [fullscreen, setFullscreen] = useState(false);
   const [isImagedLoaded, setIsImageLoaded] = useState(false);
   const isSmallerThanPhone = useBreakpoint('phone');
+  const fullscreenButtonRef = useRef<HTMLButtonElement | null>(null);
 
-  const toggleFullscreen = () => {
+  const toggleFullscreen = (event: any) => {
```

Comment:
We could use a MouseEvent here
```suggestion
  const toggleFullscreen = (event: React.MouseEvent) => {
```